cms/static/js/views/paging_header.js

```diff
@@ -31,17 +31,37 @@ define(["underscore", "gettext", "js/views/baseview"], function(_, gettext, Base
         },
 
         messageHtml: function() {
-            var message;
-            if (this.view.collection.sortDirection === 'asc') {
-                // Translators: sample result: "Showing 0-9 out of 25 total, sorted by Date Added ascending"
-                message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, sorted by %(sort_name)s ascending');
-            } else {
-                // Translators: sample result: "Showing 0-9 out of 25 total, sorted by Date Added descending"
-                message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, sorted by %(sort_name)s descending');
+            var message = '';
+            if (this.view.collection.assetType) {
+                if (this.view.collection.sortDirection === 'asc') {
+                    // Translators: sample result:
+                    // "Showing 0-9 out of 25 total, filtered by Images, sorted by Date Added ascending"
+                    message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, ' +
```

Comment:
Has the i18n been verified for this yet? I can never remember whether split lines work confuse the scraper or not.